db/index.js

```diff
@@ -1,5 +1,5 @@
 const mongoose = require('mongoose');
-const dockerDB = '172.17.0.3'; //replace localhost with when deploying
+const dockerDB = '172.17.0.3'; //replace mongo with when deploying
```

Comment:
I'm not sure what the change for this line is for...

Reply:
oh that was a note to myself since i changed it so I could see her page loaded. When it was pushed last it was setup for ec2 side (like my own)